src/ext-test/resources/legacygraphqlapi/expectations/plan.json

```diff
@@ -88,7 +90,41 @@
                     }
                   ]
                 }
-              ]
+              ],
+              "rideHailingEstimate" : null
+            },
+            {
+              "mode" : "CAR",
+              "from" : {
+                "name" : "D",
+                "lat" : 8.0,
+                "lon" : 9.5,
+                "departureTime" : 1580644200000,
+                "arrivalTime" : 1580644200000
+              },
+              "to" : {
+                "name" : "E",
+                "lat" : 9.0,
+                "lon" : 10.0,
+                "departureTime" : 1580644800000,
+                "arrivalTime" : 1580644800000
+              },
+              "startTime" : 1580644200000,
+              "endTime" : 1580644800000,
+              "generalizedCost" : 1000,
+              "alerts" : [ ],
+              "rideHailingEstimate" : {
+                "provider" : "UBER",
```

Comment:
Existing UI for TNC rides supports displaying a product name (e.g. "UberX") as part of waiting instructions. [See for instance here](http://www.opentripplanner.org/otp-ui/?path=/story/itinerarybody-otp-react-redux--tnc-transit-itinerary&globals=locale:en-US). The product name is being returned from the API call, it would be nice to continue displaying it after changes from this PR.

Reply:
cc @demory

Reply:
Good point, the information is already available in the system and can be exposed with not a lot of work.